index.html

```diff
@@ -0,0 +1,2378 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Newsweek - News, Analysis, Politics, Business, Technology</title>
+    <link rel="icon" type="image/png" sizes="32x32"
+        href="https://g.newsweek.com/themes/newsweek/favicons/favicon-32x32.png">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+    <!-- Google fonts -->
+    <link
+        href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300&family=Open+Sans:wght@300&family=Oswald:wght@200&family=Roboto+Condensed:ital,wght@0,700;1,400;1,700&display=swap"
+        rel="stylesheet">
+    <!-- Fontawesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" />
+    <link rel="stylesheet" href="assets/css/reset.css">
+    <link rel="stylesheet" href="assets/css/main.css">
+</head>
+
+<body>
+
+    <!-- Navigatio bar -->
+    <header class="nav-header">
+        <div class="container-fluid">
+            <div class="header-top d-flex justify-content-between">
+                <div class="weather-info flex-column justify-content-between d-none d-lg-flex">
+                    <div class="weather-info-instant text-white">
+                        <span class="cloud">
+                            <i class="fas fa-cloud text-white"></i>
+                        </span>
+                        <span class="weather-temp">
+                            &nbsp;&nbsp;25°
+                        </span>
+                        <span class="weather-location">
+                            Addis Ababa, ET
+                        </span>
+                    </div>
+                    <div class="date-info text-white">
+                        <time datetime="2020-05-27">Wed, May 20, 2020</time>
+                    </div>
+                </div>
+                <div class="header-logo">
+                    <a class="brand" href="https://www.newsweek.com">
+                        <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg"
+                            xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 395.6 60"
```

Comment:
[REQUIRED] We have to fix all the error shown by the HTML validator. I am using [this](https://validator.w3.org) validator.

Reply:
- HTML validator shows up the following result:  
![error-check](https://user-images.githubusercontent.com/12418320/83324767-335d2280-a270-11ea-98a9-a9be6782c7e3.png)